meson_options.txt

```diff
@@ -14,6 +14,8 @@ option('docs', type: 'feature', value: 'auto',
     description: 'Build documentation')
 option('omf-byte-order', type: 'combo', choices: ['native', 'big', 'little'],
     value: 'native', description: 'On-media-format byte order')
+option('key-cpe', type: 'combo', choices: ['speed', 'size'],
+    value: 'size', description: 'Optimize key common-prefix-elimination for size or speed')
```

Comment:
What is the speed improvement?

Reply:
I may have conflated it with LTO, lemme run some new tests...

Reply:
load: 1.28x
test read-only: 1.024x
test 20% write: 1.15x

Reply:
Do you have any stats on size optimization too? Seems like the speed optimization makes sense to have.

Reply:
I looked at a few tests, the "speed" option was 2.2G larger KWLEN on a 2-billion key kmt simulation of ycsb.
So it varies wildly on the key length and common prefix length.  If you had 9-byte common prefixes and used the "speed" option, it would waste one byte per key whereever CPE is applicable.  Worst case would be 15-byte common prefix where you'd waste 7-bytes per key.  So on average it costs about 3.5 bytes per key.
By "waste" I mean the amount of storage you could have saved with the "size" option.